$NetBSD$

From d3f46e3672462c386b0c5929e370948090bf8c30 Mon Sep 17 00:00:00 2001
From: Scott Jaderholm <jaderholm@gmail.com>
Date: Tue, 11 Apr 2017 23:29:18 -0700
Subject: [PATCH] search-engines: Update some engines to https

Updates for creativecommons, wiktionary, scholar, and slang.

Patch provided by Adam Plaice.
---
 modules/webjump.js                 | 4 ++--
 search-engines/creativecommons.xml | 5 ++---
 search-engines/wiktionary.xml      | 6 +++---
 3 files changed, 7 insertions(+), 8 deletions(-)

diff --git a/modules/webjump.js b/modules/webjump.js
index e70d167..86a45f2 100644
--- modules/webjump.js
+++ modules/webjump.js
@@ -220,8 +220,8 @@ function define_default_webjumps () {
                    "http://conkeror.org/?action=fullsearch&context=60&value=%s&fullsearch=Text");
     define_webjump("lucky",      "https://www.google.com/search?q=%s&btnI=I'm Feeling Lucky");
     define_webjump("maps",       "https://maps.google.com/?q=%s");
-    define_webjump("scholar",    "http://scholar.google.com/scholar?q=%s");
-    define_webjump("slang",      "http://www.urbandictionary.com/define.php?term=%s");
+    define_webjump("scholar",    "https://scholar.google.com/scholar?q=%s");
+    define_webjump("slang",      "https://www.urbandictionary.com/define.php?term=%s");
     define_webjump("dictionary", "http://dictionary.reference.com/search?q=%s");
     define_webjump("image",      "https://www.google.com/images?q=%s");
 }
diff --git a/search-engines/creativecommons.xml b/search-engines/creativecommons.xml
index 7baf3b0..439973c 100644
--- search-engines/creativecommons.xml
+++ search-engines/creativecommons.xml
@@ -3,9 +3,8 @@
 <Description>Find photos, movies, music, and text to rip, sample, mash, and share.</Description>
 <InputEncoding>utf-8</InputEncoding>
 <Image width="16" height="16">data:image/x-icon;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAAN1wAADdcBQiibeAAAABl0RVh0U29mdHdhcmUAd3d3Lmlua3NjYXBlLm9yZ5vuPBoAAAJUSURBVDiNjZO9S1thGMXPvTfJFbnkipNkLLS0ksFg0M0lf4CboNCEgIIg/RiKtEOn0qGWTtbVoBYcIji10I9J0ApWCjp0kRaXdhHjTW4+uGnur4NJ8GOwZ3nf4TnnfZ5z3scAdBGGYdyVdN+yrGHTNNOtVqsVhuG+pO+S3gE/LtV3BIxzPDJNc8FxHGN0dNRKpVIGoJ2dndr+/r5Vr9cl6bmkN0AoSQIEyHXdj5KYnZ3F932uolKpkM/nK5KQ9FmSCZwLOI7zQBLr6+vXiFdRLBaDtsiTTve3YrFYkM/nbyR3MDU1dSKpLumO+vr6Xruui+d5AFSrVVZWVtjY2KDRaABwdHTE4uIie3t7AJTLZaLRaFXSCyUSid1MJgOA53n09/eTTqdJJpPMzc2xurqKbduMj48Tj8fZ3d0FYHBw8FjSezmOU56fnwdgeXkZ27ap1WpUKhWazSZjY2Nks1kASqVSd4zp6eljSX/MtiHdRDpnEATyfb+bkiSVSqXu3TCM8xgHBga+dkY4OzvDdV2GhoZIJBLMzMxQKBSIRqNkMhlisRhbW1sAJJPJn5I+KB6Pv7poou/7rK2tsbm5SRAEXROXlpY4ODgAoFarYdu2J+llN8ZcLvffMeZyud+SGpLuCVBPT89jSRQKhRvJxWKxISmU9JTOT5Rk9Pb2fpHE5OQkJycn14inp6dMTEx4bdM/SbKAy8sk6WEkElmwLCuSSqUYGRmxgHB7e7t+eHgYazabgaRnkt7SeZnr63xbUtYwjGHTNNNhGP4F9iR9a6/zr4v1/wDE1D9XlC4rrAAAAABJRU5ErkJggg==</Image>
-<Url type="text/html" method="GET" template="http://search.creativecommons.org/">
+<Url type="text/html" method="GET" template="https://search.creativecommons.org/">
   <Param name="q" value="{searchTerms}"/>
-  <Param name="sourceid" value="Mozilla-search"/>
 </Url>
-<SearchForm>http://search.creativecommons.org/</SearchForm>
+<SearchForm>https://search.creativecommons.org/</SearchForm>
 </SearchPlugin>
diff --git a/search-engines/wiktionary.xml b/search-engines/wiktionary.xml
index 6b5ca91..91c7f68 100644
--- search-engines/wiktionary.xml
+++ search-engines/wiktionary.xml
@@ -3,13 +3,13 @@
 <Description>Wiktionary, a wiki-based Open Content dictionary</Description>
 <InputEncoding>UTF-8</InputEncoding>
 <Image width="16" height="16">data:image/x-icon;base64,AAABAAEAEBAQAAEABAAoAQAAFgAAACgAAAAQAAAAIAAAAAEABAAAAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAEAgQAhIOEAMjHyABIR0gA6ejpAGlqaQCpqKkAKCgoAPz9%2FAAZGBkAmJiYANjZ2ABXWFcAent6ALm6uQA8OjwAiIiIiIiIiIiIiI4oiL6IiIiIgzuIV4iIiIhndo53KIiIiB%2FWvXoYiIiIfEZfWBSIiIEGi%2FfoqoiIgzuL84i9iIjpGIoMiEHoiMkos3FojmiLlUipYliEWIF%2BiDe0GoRa7D6GPbjcu1yIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA</Image>
-<Url type="application/x-suggestions+json" method="GET" template="http://en.wiktionary.org/w/api.php">
+<Url type="application/x-suggestions+json" method="GET" template="https://en.wiktionary.org/w/api.php">
   <Param name="action" value="opensearch"/>
   <Param name="search" value="{searchTerms}"/>
 </Url>
-<Url type="text/html" method="GET" template="http://en.wiktionary.org/wiki/Special:Search">
+<Url type="text/html" method="GET" template="https://en.wiktionary.org/wiki/Special:Search">
   <Param name="search" value="{searchTerms}"/>
   <Param name="sourceid" value="Mozilla-search"/>
 </Url>
-<SearchForm>http://en.wiktionary.org/wiki/Special:Search</SearchForm>
+<SearchForm>https://en.wiktionary.org/wiki/Special:Search</SearchForm>
 </SearchPlugin>
-- 
2.7.4.GIT

